src/components/dashboard/KeyPerformanceIndicator.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="kpi-container mx-3">
+    <div class="centered">
```

Comment:
can these div's be collapsed into one?

Reply:
`has-text-centered` can probably be moved out to one of these containers instead of repeated

Reply:
Yes! I've cleaned up the markup here